### DISCUSSION TITLE
How to make the "move" command move across the monitor

### DISCUSSION THREAD

**Discussion by mortyhuang** [nikitabobko/AeroSpace#881] [feature-ideas]
Is it possible to add parameters similar to "focus" to "move"?
Just like this.
https://github.com/nikitabobko/AeroSpace/discussions/654#discussioncomment-11550329

**Comment by gldtn** [comment]
Hi, I don't think it's possible yet. There is a open issue talking about here: https://github.com/nikitabobko/AeroSpace/issues/183

Also a PR  here: https://github.com/nikitabobko/AeroSpace/pull/864

**Reply by mortyhuang** [reply]
Thanks for the info. I'll keep an eye on the issue for updates.

In [glazewm](https://github.com/glzr-io/glazewm), which I used previously, the move command defaulted to moving workspaces across multiple screens, simplifying the need for numerous keyboard shortcuts. I found this quite useful.
